changelog/bE5Htd8cQja6Jzk722vy1A.md

```diff
@@ -0,0 +1,4 @@
+audience: general
+level: patch
+---
+Generic Worker: No longer logs out the redirect URL for Redirect Artifacts whose name begins with `private/`.
```

Comment:
let's just change this to say that we don't log the url of interactive artifacts, since they are signed urls... something like that